scrapy/spiders/__init__.py

```diff
@@ -66,10 +66,20 @@ def _set_crawler(self, crawler):
         crawler.signals.connect(self.close, signals.spider_closed)
 
     def start_requests(self):
-        for url in self.start_urls:
-            yield self.make_requests_from_url(url)
+        if self.make_requests_from_url is not Spider.make_requests_from_url:
```

Comment:
^^ this line is wrong because the condition is always True: bound method is not the same as unbound.